dae/dae/annotation/tests/test_pipeline.py

```diff
@@ -32,3 +38,32 @@ def test_build_pipeline_schema(
     assert "cadd_raw" in schema.names
     field = schema["cadd_raw"]
     assert field.type == "float"
+
+
+@pytest.mark.xfail()
+def test_internal_attribute_filtered(
+        phastcons100way_indel_variants_expected,
+        grr_fixture):
+    resource = grr_fixture.get_resource("hg38/TESTphastCons100way")
+
+    config = {
+        "annotator_type": "position_score",
+        "resource_id": "hg38/TESTphastCons100way",
+        "attributes": [{
+            "source": "phastCons100way",
+            "destination": "phastCons100way",
+            "position_aggregator": "mean",
+            "internal": True
+        }]
+    }
+    score = open_position_score_from_resource(resource)
+
+    annotator = PositionScoreAnnotator(config, score)
+    pipeline = AnnotationPipeline([], grr_fixture)
+    pipeline.add_annotator(annotator)
+
+    variant = phastcons100way_indel_variants_expected[0][0]
+    allele = variant.alt_alleles[0]
+
+    result = pipeline.annotate(allele.get_annotatable())
+    assert "phastCons100way" not in result
```

Comment:
* "phastCons100way" should be present in the resulting allele;
* if you check the pipeline schema fields, the "phasCons100way" should by found in internal_fields, and should be missing in public_fields.